src/drive/web/modules/actionmenu/ActionsItems.jsx

```diff
@@ -7,20 +7,28 @@ import styles from 'drive/styles/actionmenu.styl'
 export const ActionsItems = ({ actions, file, onClose }) => {
   const { t } = useI18n()
   let previousAction = ''
-  return actions.map((actionObject, i) => {
-    const actionName = Object.keys(actionObject)[0]
+  const displayableActions = actions.filter(actionObject => {
     const actionDefinition = Object.values(actionObject)[0]
-    if (
+    return !(
```

Comment:
Can you convert the boolean to a positive one ? the `!` + `!==` + `&&` makes it hard to understand.

Reply:
I tried to find something easier, but without any success. If you have an idea ... 👍 

Will add a test for those methods

Reply:
```
   return !actionDefinition.displayCondition || actionDefinition.displayCondition([file])
```
?

Reply:
thanks!